zmon-controller-app/src/main/resources/templates/index.html

```diff
@@ -109,6 +109,20 @@
           }
     </script>
 
+    <!-- OpenTracing -->
+    <script th:inline="javascript" th:if="${eumTracingEnabled}" src="/lib/opentracing-javascript/opentracing-browser.min.js"></script>
+    <script th:inline="javascript" th:if="${eumTracingEnabled}" src="/lib/opentracing-javascript-utils/opentracing-javascript-utils.js"></script>
+    <script th:inline="javascript" th:if="${eumTracingEnabled}">
+      var eumTracingConfig = /*[[${eumZmonTracingConfig}]]*/  { name: "noop", config: {} };
+      eumTracingConfig = JSON.parse(eumTracingConfig);
```

Comment:
If you provide an object `Map<String, Object>` instead of simple string, this parsing can be removed.